featurebyte/schema/info.py

```diff
@@ -504,3 +504,15 @@ class UseCaseInfo(BaseInfo):
     target_name: str
     default_eda_table: Optional[str] = None
     default_preview_table: Optional[str] = None
+
+
+class ContextInfo(BaseInfo):
+    """
+    Context Info schema
+    """
+
+    author: Optional[str] = None
+    entities: List[str]
```

Comment:
should this be of the `EntityBriefInfoList` type, some of the other info structs also use `EntityBriefInfoList`